src/ProofOfConcepts/ACLFilterChips/OneGroup.tsx

```diff
@@ -0,0 +1,16 @@
+import { VoidFunctionComponent } from "react";
+import React from "react";
+import Toolbar from "./toolbarOneGroup";
+import Chips from "./optionOneGroup";
+
+export const OneGroupChips: VoidFunctionComponent = () => {
+  return (
+    <React.Fragment>
+      {/* Mock toolbar with options pre-checked*/}
+      <Toolbar />
+      <br></br>
+      &emsp;
```

Comment:
I don't think we use `<br>` elements when building UI. Usually, any spacing needed is handled with CSS. PF usually provides the spacing we need—a component will define space it needs inside the component, and a layout will define the space between components. Ideally, PF options for modifying spacing are what we use, but sometimes there are designs that require some customization.

For this layout, I think the Toolbar component provides what's needed to handle layout and spacing. There is this example showing a toolbar with filters that you should be able to follow and use here: https://www.patternfly.org/v4/components/toolbar#with-filters